config/routes.rb

```diff
@@ -19,6 +19,12 @@
   constraints Constraints::LocalAuthoritySubdomain do
     root to: "planning_applications#index"
 
+    concern :positionable do |options|
+      defaults format: "json" do
+        resource :position, {only: %i[update]}.merge(options)
```

Comment:
Uses `:position` and not `:positions` since it's an action on a singular item in the collection